cli/src/commands/build_docker_scraper.py

```diff
@@ -9,12 +9,13 @@ def get_description(self):
         return 'Build scraper images (dev, prod)'
 
     def run(self, args):
-        py3 = self.get_option('python3', args)
 
-        code = self.build_docker_file("scraper/dev/docker/Dockerfile.base", "algolia/base-documentation-scrapper")
+        code = self.build_docker_file("scraper/dev/docker/Dockerfile.base",
+                                      "algolia/base-documentation-scraper")
```

Comment:
Shouldn't we rename it to `algolia/docsearch-base`?

Reply:
I miss the docsearch rename yes

Reply:
Actually it is solved